pkg/dockerfile/generator_test.go

```diff
@@ -37,16 +37,20 @@ func getWheelName() string {
 	return files[0].Name()
 }
 
-func testInstallCog(relativeTmpDir string, stripped bool) string {
+func testInstallCog(relativeTmpDir string, stripped bool, baseImage bool) string {
 	wheel := getWheelName()
 	strippedCall := ""
 	if stripped {
 		strippedCall += " && find / -type f -name \"*python*.so\" -not -name \"*cpython*.so\" -exec strip -S {} \\;"
 	}
+	baseImageArgs := " -t /dep"
```

Comment:
What's the scenario in which `-t /dep` is needed?

Reply:
There is a function called [copyPipPackagesFromInstallStage](https://github.com/replicate/cog/blob/main/pkg/dockerfile/generator.go#L535) which is executed at the end of an install that seems to copy a bunch of packages from `/dep` directly into the pyenv site-packages. It's unclear to me why this is done like this, I suspect it might be due to wanting to put these deps into both the system python and the pyenv python.

Reply:
Hmm. That sounds like something we should probably understand and, if possible, simplify.